drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -186,7 +183,7 @@ def studio_view(self, context):
 
         js_templates = loader.load_unicode('/templates/html/js_templates.html')
         help_texts = {
-            field_name: self._(field.help)
+            field_name: self.ugettext(field.help)
```

Comment:
For just testing, you can mark any string(s) in `student_view` with `self.ugettext`. You will get the translated string(s)